src/aws-ec2/README.md

```diff
@@ -0,0 +1,78 @@
+Constructs for the AWS EC2 service
+
+# EC2 Instance Connect Endpoint CDK Construct
+
+## Overview
+
+The `InstanceConnectEndpoint` construct facilitates the creation and management of [EC2 Instance Connect endpoints](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/connect-with-ec2-instance-connect-endpoint.html)
+within AWS CDK applications.
+
+## Usage
+
+Import the necessary classes from AWS CDK and this construct and create a VPC for the endpoint:
+
+```ts
+import { App, Stack } from 'aws-cdk-lib';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import { InstanceConnectEndpoint } from '@open-constructs/aws-cdk/aws-ec2';
+
+const app = new App();
+const stack = new Stack(app, 'InstanceConnectEndpointStack');
+const vpc = new ec2.Vpc(stack, 'MyVpc');
+```
+
+### Basic Example
+
+Here's how you can create an EC2 Instance Connect endpoint and allow connections to an EC2 instance:
+
+```ts
+const instance = new cdk.aws_ec2.Instance(this, 'Instance', {
+  vpc,
+  instanceType: cdk.aws_ec2.InstanceType.of(
+    cdk.aws_ec2.InstanceClass.C5,
+    cdk.aws_ec2.InstanceSize.LARGE,
+  ),
+  machineImage: new cdk.aws_ec2.AmazonLinuxImage({
+    generation: cdk.aws_ec2.AmazonLinuxGeneration.AMAZON_LINUX_2023,
+  }),
+});
+
+const endpoint = new InstanceConnectEndpoint(stack, 'MyEndpoint', {
+  vpc,
+});
+
+// Allow SSH connections to the instance
+// You can also use the port 3389 for RDP connections
+endpoint.connections.allowTo(instance, ec2.Port.tcp(22));
+```
```

Comment:
Oops. Let's put this in here.

```ts
import { InstanceConnectEndpoint } from '@open-constructs/aws-cdk/aws-ec2';
```

Reply:
Is it ok because it's in the Usage? But I think I get a rosetta error with aws-cdk. Which one is correct...?

Reply:
If we have any concerns at all, I feel it's better to define them here, just in case...?

Reply:
@badmintoncryer What do you think?  We would be happy to provide these Constructs to aws-cdk.

Reply:
Actually let's just go with it.

Reply:
I unified the expressions such as cdk.aws_ec2.Xxx and ec2.Xxx to consistently use ec2.Xxx.

Reply:
In the readme, I mistakenly thought that defining variables above the specified section would prevent errors in Rosetta. 
By the way, is the readme CI not running in the aws-cdk-library now?

Reply:
Thanks for the changes. CI has not yet run linter, etc.